### PR TITLE
Testing Hooks and Components fixes

### PR DESCRIPTION
**Pull request by vasilii-kovalev**
* `INSTRUCTIONS.md`
  * Remove `userEvent` from `@testing-library/react` import (because it is not exported from there) and add a standalone one.
* `use-async.extra-1.js`
  * Fix a typo

### CODE REVIEW

**Review by JacobMGEvans — APPROVED** [review]

**Review by JacobMGEvans — APPROVED** [review]

File: src/utils/__tests__/use-async.extra-1.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ test('calling run with a promise which rejects', async () => {
   await act(async () => {
     reject(rejectedValue)
     await p.catch(() => {
-      /* ignore erorr */
+      /* ignore error */
```
Comment (JacobMGEvans) [inline src/utils/__tests__/use-async.extra-1.js:96]: Nice catch! 😄

**Review by kentcdodds — APPROVED** [review]
Thanks! This has been fixed https://github.com/kentcdodds/bookshelf/blob/main/CONTRIBUTING.md#read-this-please

**Comment by kentcdodds** [comment]
Whoops. I merged this 🤦‍♂️

**Comment by kentcdodds** [comment]
Ok, luckily I had the correct commit locally and could force-push 😅

**Pull request merged**